basic/ncr.py

```diff
@@ -0,0 +1,16 @@
+def nCr(n, r):
+    return (fact(n) / (fact(r)* fact(n - r)))
+
+
+# Returns factorial of n
+def fact(n):
+    res = 1
+
+    for i in range(2, n + 1):
+        res = res * i
+
+    return res
+# Driver code
```

Comment:
I don't think we're including driver code in this repo since "once all the functions are coded, main.py will be converted into a menu-driven program".  Having extra code here may interfere with importing later (unintended side-effects).  If maintainer decides this is desirable, we may still want to guard on `__name__ == '__main__'`